backend/tests/apps/github/management/commands/github_match_users_test.py

```
@@ -0,0 +1,274 @@
+        exact, fuzzy, unmatched = command.process_leaders(leaders_raw, 75, mock_users)
+        
+        assert len(exact) == 2
+        assert mock_users[1] in exact
+        assert mock_users[2] in exact
+        assert fuzzy == []
+        assert unmatched == []
+
+    @patch("apps.github.management.commands.github_match_users.fuzz")
+    def test_fuzzy_match(self, mock_fuzz, command, mock_users):
+        """Test fuzzy matching."""
+        mock_fuzz.token_sort_ratio.side_effect = lambda l, r: 90 if "peter" in r.lower() or "peter" in l.lower() else 10
```

Ruff (0.11.9)

Ambiguous variable name: `l`

(E741)